bin/citrea/tests/bitcoin_e2e/light_client_test.rs

```diff
@@ -143,3 +149,315 @@ async fn test_light_client_proving() -> Result<()> {
         .run()
         .await
 }
+
+struct LightClientProvingTestMultipleProofs;
+
+#[async_trait]
+impl TestCase for LightClientProvingTestMultipleProofs {
+    fn test_config() -> TestCaseConfig {
+        TestCaseConfig {
+            with_sequencer: true,
+            with_batch_prover: true,
+            with_light_client_prover: true,
+            with_full_node: true,
+            ..Default::default()
+        }
+    }
+
+    fn sequencer_config() -> SequencerConfig {
+        let mut cfg = SequencerConfig {
+            min_soft_confirmations_per_commitment: 20,
+            da_update_interval_ms: 500,
+            ..Default::default()
+        };
+
+        cfg.mempool_conf.max_account_slots = 400;
+        cfg
+    }
+
+    fn batch_prover_config() -> BatchProverConfig {
+        BatchProverConfig {
+            enable_recovery: false,
+            ..Default::default()
+        }
+    }
+
+    fn light_client_prover_config() -> LightClientProverConfig {
+        LightClientProverConfig {
+            enable_recovery: false,
+            ..Default::default()
+        }
+    }
+
+    async fn run_test(&mut self, f: &mut TestFramework) -> Result<()> {
+        let da = f.bitcoin_nodes.get(0).unwrap();
+        let sequencer = f.sequencer.as_ref().unwrap();
+        let batch_prover = f.batch_prover.as_ref().unwrap();
+        let light_client_prover = f.light_client_prover.as_ref().unwrap();
+        let full_node = f.full_node.as_ref().unwrap();
+
+        let min_soft_confirmations_per_commitment =
+            sequencer.min_soft_confirmations_per_commitment();
+
+        let rpc_bind_host = sequencer.config().rpc_bind_host();
+        let rpc_bind_port = sequencer.config().rpc_bind_port();
+
+        let socket_addr = SocketAddr::new(rpc_bind_host.parse().unwrap(), rpc_bind_port);
+
+        let seq_test_client = make_test_client(socket_addr).await.unwrap();
+
+        // publish min_soft_confirmations_per_commitment confirmations
+        for _ in 0..4 * min_soft_confirmations_per_commitment {
```

Comment:
```suggestion
        for _ in 0..min_soft_confirmations_per_commitment * n_conf {
```
Can lift 4 to a var and re-use on the * 4 and 8 below